app/helpers/responses_helper.rb

```diff
@@ -0,0 +1,26 @@
+module ResponsesHelper
```

Comment:
passing the response to each of these methods is similar to existing project_step_helper. Happy for suggestions about making it more concise